modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppPistacheServerCodegen.java

```diff
@@ -250,7 +250,7 @@ public CodegenOperation fromOperation(String path, String httpMethod, Operation
                 if(importMapping.containsKey(hdr)) {
                     continue;
                 }
-                additionalProperties.put("hasModelImport", true);
+                operations.put("hasModelImport", true);
```

Comment:
wouldnt it better to put this in the vendor extensions?

Reply:
We may consider adding this to the top level so that all generators can also use `hasModelImport` (I've seen another related request before)

Reply:
@etherealjoy please go with vendor extension for the time being (e.g. x-codegen-pistache-hasModelImport)

Reply:
@wing328 @stkrwork 
Ok, done using vendor extensions